docs/content/_navigation.json

```diff
@@ -6,28 +6,20 @@
     "isDefaultOpen": true,
     "children": [
       {
-        "title": "Hello Dagster",
+        "title": "Overview",
+        "path": "/getting-started/overview"
+      },
+      {
+        "title": "Quickstart",
         "path": "/getting-started/hello-dagster"
       },
       {
         "title": "Installation",
         "path": "/getting-started/install"
       },
-      {
-        "title": "Create a new project",
-        "path": "/getting-started/create-new-project"
-      },
-      {
-        "title": "Project files",
-        "path": "/getting-started/project-file-reference"
-      },
       {
         "title": "Getting help",
         "path": "/getting-started/getting-help"
-      },
-      {
-        "title": "Telemetry",
```

Comment:
I think this would make more sense under "About". The main reason for this page is being transparent about the fact that we collect telemetry, rather than guiding the user on how to do something.

Reply:
Works for me! Moved it.